pinax/stripe/actions/customers.py

```diff
@@ -39,6 +39,20 @@ def create(user, card=None, plan=settings.PINAX_STRIPE_DEFAULT_PLAN, charge_imme
     Returns:
         the pinax.stripe.models.Customer object that was created
     """
+    try:
+        cus = models.Customer.objects.get(user=user)
+    except models.Customer.DoesNotExist:
+        pass
+    else:
+        try:
+            stripe.Customer.retrieve(cus.stripe_id)
+        except stripe.error.InvalidRequestError:
+            pass
+        else:
+            return cus
+
+    # At this point we maybe have a local Customer but no stripe customer
+    # let's create one and make the binding
```

Comment:
You said:

"We create and then delete a remote customer" (my paraphrase) in https://github.com/pinax/pinax-stripe/pull/402#issuecomment-338257188

So are you trying to just recreate the remove customer you created and then deleted?  Are you doing the create/delete manually in the Stripe dashboard?

I think there is another PR that is merged or is to be merged that handles the `delete` webhook and purges the customer, removing the `user` from the `Customer` by setting it to None.  In which case, I'm guessing this will just create a new one.